src/modules/attributes.ts

```diff
@@ -48,4 +51,4 @@ function updateAttrs(oldVnode, vnode) {
   }
 }
 
-module.exports = {create: updateAttrs, update: updateAttrs};
+export = {create: updateAttrs, update: updateAttrs} as Hooks;
```

Comment:
This, although it overlaps, should be exported `as Module` instead of `as Hooks`.